pages/json.md

```diff
@@ -0,0 +1,52 @@
+#JSON
+
+##JSON implementation in Rust:
+
+
+[![json][json-badge]][json]
+
+```rust
+#[macro_use]
+extern crate json;
+
+fn main() {
+
+let parsed = json::parse(r#"
+
+{
+    "code": 200,
+    "success": true,
+    "payload": {
+        "features": [
+            "awesome",
+            "easyAPI",
+            "lowLearningCurve"
+        ]
+    }
+}
+
+"#).unwrap();
+
+let instantiated = object!{
+    "code" => 200,
+    "success" => true,
+    "payload" => object!{
+        "features" => array![
+            "awesome",
+            "easyAPI",
+            "lowLearningCurve"
+        ]
+    }
+};
+
+assert_eq!(parsed, instantiated);
```

Comment:
Rather than comparing the object is equal, please show how to traverse the JSON document to compare individual pieces of data.